style.css

```diff
@@ -462,16 +462,248 @@ footer {
 }
 
 .Summit-text span {
-  text-align: center;
   font-family: 'Lato', sans-serif;
+  text-align: center;
   color: #fff;
   font-size: 24px;
   font-weight: 600;
-  z-index: 1
+  z-index: 1;
 }
 
 /* Mobile About Page */
 
+/* Desktop Home Page */
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] To improve the user interface of your site, it would be great if you could fix the responsiveness of your page as it breaks from `768px` to `1600px` as shown in the video below.
- Your site:
![Animation](https://user-images.githubusercontent.com/85113567/170286982-75776c96-7467-4e0f-9166-e3474f353ff1.gif)